.github/workflows/propagate_to_osbuild.yml

```diff
@@ -0,0 +1,63 @@
+name: Propagate update to OSBuild
+
+on: workflow_dispatch
+
+jobs:
+  propagate:
+    runs-on: ubuntu-latest
+    env:
+      SCHUTZBOT_GH: ${{ secrets.SCHUTZBOT_GH}}
+    steps:
+      - name: Checkout manifest-db
+        uses: actions/checkout@v3
+        with:
+          path: manifest-db
+          fetch-depth: 0
+
+      - name: Update OSBuild
+        run: |
+          GH_REF="${{github.ref}}"
+          if [[ "$GH_REF" == *main* ]] ; then
+            # Get
+            MANIFEST_DB_HEAD="${{ github.sha }}"
+            echo "SHA: $MANIFEST_DB_HEAD"
+
+            # login as schutzbot
+            echo "${SCHUTZBOT_GH}" | gh auth login --with-token
+
+            # Clone osbuild
+            git clone https://github.com/osbuild/osbuild.git
+
+            cd osbuild
+            git config --local user.name "SchutzBot"
+            git config --local user.email "imagebuilder-bots+schutzbot@redhat.com"
+
+            git remote add upstream https://schutzbot:"$SCHUTZBOT_GH"@github.com/schutzbot/osbuild.git
+
+            now=$(date '+%Y-%m-%d-%H%M%S')
+            BRANCH_NAME="manifest-db-update-$now"
+            git checkout -b $BRANCH_NAME
+
+            # get the old ref from the OSBuild's Schutzfile
+            OLD_REF=$(jq -r '.global.dependencies."manifest-db".commit' Schutzfile)
+
+            # update only if the DB was updated since the version OSBuild is
+            # using
+            cd ../manifest-db
+            if [[ "$(git diff --name-only $GH_REF $OLD_REF manifest-db)" == *manifest-db* ]]; then
+              cd ../osbuild
+              # change the value for the commit head in the schutzfile
+              jq --arg variable "$MANIFEST_DB_HEAD" '.global.dependencies."manifest-db".commit=$variable' Schutzfile > Schutzfile.tmp && mv Schutzfile.tmp Schutzfile
+
+              git add -A && \
+                  git commit -m "schutzfile: update manifest-db ref" && \
+                 git push upstream "$BRANCH_NAME:$BRANCH_NAME" && \
+                  gh pr create \
+                      --title "schutzfile: update manifest-db ref" \
+                      --body "automated update." \
```

Comment:
What about putting the list of commits between `$GH_REF` and `$OLD_REF` into the body? An output of `git log --oneline $OLD_REF..$GH_REF` might be the right thing.

Reply:
Good idea! I was short of ideas on how to fill up the body message. The list of commits can do the job indeed.